app/javascript/components/pages/EventShow.js

```diff
@@ -1,6 +1,40 @@
 import React, { Component } from 'react';
 
 class EventShow extends Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      loading: true,
+      event: false,
```

Comment:
```suggestion
      event: undefined,
```
initializing values for state properties should always be of the same data type as the actual data or undefined